src/components/Scrollable/Scrollable.tsx

```diff
@@ -131,7 +134,7 @@ export class Scrollable extends Component<ScrollableProps, State> {
             {...rest}
             ref={this.setScrollArea}
             // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
-            tabIndex={0}
+            tabIndex={focusable ? 0 : -1}
```

Comment:
I think in this case we might want `undefined` since -1 is a thing. I needed something similar here: https://github.com/Shopify/polaris-react/pull/3595/files#diff-3a6c8eaf1de81279fa78d47ad8cd27a34087c7c29881ac4c9a1c881858132c08L214

For context on that change I linked to... when inside a Combobox/autocomplete pattern, we can't have anything steal focus from the text field. In the current implementation, we watch for it and then set the focus back on the field if it wasn't intentional. That results in a poor ux for screen reader users as the lost and gain of focus creates lots of unnecessary noise.